crates/polars-core/src/frame/explode.rs

```diff
@@ -308,7 +308,7 @@ impl DataFrame {
         let mut iter = on.iter().map(|v| {
             schema
                 .get(v)
-                .ok_or_else(|| polars_err!(ColumnNotFound: "{}", v))
+                .ok_or_else(|| polars_err!(ColumnNotFound: "{:?}", v))
```

Comment:
Can we add a `polars_err!(col_not_found = name)` shorthand in `polars_err`. Then the formatting is in one place.